model/src/main/java/org/cbioportal/genome_nexus/model/VEPInfo.java

```diff
@@ -0,0 +1,25 @@
+package org.cbioportal.genome_nexus.model;
+
+public class VEPInfo {
+
+    private final String NA_STRING = "NA";
+    private final String COMMENT = "VEP annotations are currently externally sourced from ENSEMBL. Results are subject to change without notice.";
+
+    // server corresponds to genome nexus VEP code release
+    // cache corresponds to downloaded FASTA cache
+    // comment is an additional disclaimer in the event we are using ENSEMBL VEP
+    public Version server;
```

Comment:
The more standard form would be private data members with get()/set() accessor functions. (Comment applies to GenomeNexusInfo.java too). With this form, we may see (in other classes' member functions) things like "returnedVepInfo.cache = updatedVersion;" instead of "returnedVepInfo.setCache(updatedVersion);"
This is not really a big deal unless at some point down the road we want to alter the implementation of VEPInfo to something with a different data representation (which is unlikely)